docs/resources/Guild_Scheduled_Event.md

```diff
@@ -6,23 +6,24 @@ A representation of a scheduled event in a [guild](#DOCS_RESOURCES_GUILD/).
+| privacy_level         | [privacy level](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-privacy-level)        | the privacy level of the scheduled event                                                                                                                                                                              |
+| status                | [event status](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-status)                | the status of the scheduled event                                                                                                                                                                                     |
+| entity_type           | [scheduled entity type](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-types) | the type of the scheduled event                                                                                                                                                                                       |
+| entity_id             | ?snowflake                                                                                                                     | the id of an entity associated with a guild scheduled event                                                                                                                                                           |
+| entity_metadata **    | ?[entity metadata](#DOCS_RESOURCES_GUILD_SCHEDULED_EVENT/guild-scheduled-event-object-guild-scheduled-event-entity-metadata)   | additional metadata for the guild scheduled event                                                                                                                                                                     |
+| creator?              | [user](#DOCS_RESOURCES_USER/user-object) object                                                                                | the user that created the scheduled event                                                                                                                                                                             |
+| user_count?           | integer                                                                                                                        | the number of users subscribed to the scheduled event                                                                                                                                                                 |
+| image?                | [image data](#DOCS_REFERENCE/image-data)                                                                                       | the cover image of the scheduled event                                                                                                                                                                                |
```

Comment:
```suggestion
| image?                | ?string                                                                                                                        | the [cover image hash](#DOCS_REFERENCE/image-formatting) of the scheduled event                                                                                                                                       |
```
also are you sure that this is optional?

Reply:
I am pretty sure.